doc/tutorials/quickstart_in_rviz/launch/demo.launch.py

```diff
@@ -102,16 +125,15 @@ def generate_launch_description():
         executable="spawner",
         arguments=["panda_hand_controller", "-c", "/controller_manager"],
     )
+    nodes_to_start = [
+        rviz_node,
+        static_tf,
+        robot_state_publisher,
+        run_move_group_node,
+        ros2_control_node,
+        joint_state_broadcaster_spawner,
+        arm_controller_spawner,
+        hand_controller_spawner,
+    ]
 
-    return LaunchDescription(
-        [
-            rviz_node,
-            static_tf,
-            robot_state_publisher,
-            run_move_group_node,
-            ros2_control_node,
-            joint_state_broadcaster_spawner,
-            arm_controller_spawner,
-            hand_controller_spawner,
-        ]
-    )
+    return nodes_to_start
```

Comment:
Would this have to be `return LaunchDescription(nodes_to_start)`?

Reply:
`nodes_to_start` is the return of the OpaqueFunction, which gets returned from `generate_launch_description` as:

```
    return LaunchDescription(
        declared_arguments + [OpaqueFunction(function=launch_setup)]
    )
```

Reply:
Ah yes, that would do it.